flambe/learn/train.py

```diff
@@ -219,14 +219,9 @@ def _train_step(self) -> None:
 
                 # Update iter scheduler
                 if self.iter_scheduler is not None:
-                    log(f'{tb_prefix}Training/LR', self.iter_scheduler.get_lr()[0], global_step)
-                    if isinstance(self.iter_scheduler, ReduceLROnPlateau):
-                        self.iter_scheduler.step(val_loss)
```

Comment:
Isn't this case valid anymore?

Reply:
On each iteration you can't look at the val loss really. We could expand to provide a closure based on the training loss I guess? Maybe let's keep it like this for now until the need arises

Reply:
Does this mean that we will not support `ReduceLROnPlateau`?

Reply:
We do for the normal val step schedulers but it doesn't make sense for the iter scheduler